.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@ repos:
       additional_dependencies:
       - flake8-pytest-style
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.790
+    rev: v0.800
```

Comment:
We can update pre-commit as orthogonal work. Might as well get all dependencies at once #1317